src/components/category.js

```diff
@@ -0,0 +1,9 @@
+const Category = () => (
+  <>
+    <div className="category">
+      <h1> Category</h1>
+    </div>
+  </>
```

Comment:
- [ ] Kindly update category component to display a button with 'check status'

Reply:
![image](https://user-images.githubusercontent.com/45159038/190605174-574e5875-720f-40fc-88fb-4ba1b10dbcb7.png)
Done!